src/metabase/lib/common.cljc

```diff
@@ -36,8 +36,9 @@
      {:pre [(symbol? op-name)
             (every? vector? argvecs) (every? #(every? symbol? %) argvecs)
             (every? #(not-any? #{'query 'stage-number} %) argvecs)]}
-     `(do
-          (mu/defn ~(symbol (str (name op-name) "-clause")) :- ~(keyword "mbql.clause" (name op-name))
+     (let [fn-rename #(get {"/" "div"} (name %) (name %))]
```

Comment:
`/-clause` is an invalid function name - I don't know if `/` is special cased or what.